### PR TITLE
Sticked feature: Add Item page

### PR DESCRIPTION
**Pull request by karuniasentosa**
I have put the logic inside the add item page.

### CODE REVIEW

File: lib/ui/add_item_page.dart [hunk 1]
```diff
@@ -133,3 +219,155 @@ class _AddItemPageState extends State<AddItemPage> {
     _descriptionController.dispose();
   }
 }
+
+typedef ReceiveImageCallback = void Function(File file);
+
+class _ImagePick extends StatefulWidget {
```
Comment (Gdcr7) [inline lib/ui/add_item_page.dart:225]: Everything looks good to me, but can this widget be separated to a different file? (this is not a command, but an actual question).

**Review by Gdcr7 — APPROVED** [review]
Looks good!

File: lib/ui/add_item_page.dart (hunk 1 above)
Comment (karuniasentosa) [reply lib/ui/add_item_page.dart]: All done!

File: lib/widget/icon_with_text.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class IconWithText extends StatelessWidget {
+  final Icon icon;
+  final String text;
+  final TextStyle? textStyle;
+
+  IconWithText({required this.icon, required this.text, this.textStyle});
+
+  @override
+  Widget build(BuildContext context) {
+    return Flexible(
```
Comment (Gdcr7) [inline lib/widget/icon_with_text.dart:12]: Flexible widget should be removed, flexible/expanded widgets only works inside a flex parents like column, or row.

**Review by Gdcr7 — CHANGES_REQUESTED** [review]

File: lib/widget/icon_with_text.dart (hunk 2 above)
Comment (karuniasentosa) [reply lib/widget/icon_with_text.dart]: You're absolutely right! I was worried about that error `Incorrect use of ParentDataWidget` :+1:

**Review by Gdcr7 — APPROVED** [review]

**Pull request merged**